src/cmd-run

```diff
@@ -112,6 +137,11 @@ kernel.printk = 3 4 1 7
 EOF
 )
 
+case $arch in
+    x86_64|aarch64|s390x) VM_TERMINAL="ttyS0";;
```

Comment:
Can you make `aarch64`'s terminal `"ttyAMA0"`?

Reply:
Sure, I will add that. It is the default console in the qemu, right? I haven't got around to evaluate this there yet.

Reply:
Yes. Thank you.